app/templates/user.html

```diff
@@ -10,11 +10,19 @@ <h1>User: {{ user.username }}</h1>
                 {% if user.last_seen %}<p>Last seen on: {{ moment(user.last_seen).format('LLL') }}</p>{% endif %}
                 <p>{{ user.followers.count()}} followers, {{ user.followed.count()}} following.</p>
                 {% if user == current_user %}
-                <p><a href="{{ url_for('core.edit_profile')}}">Edit Profile</a></p>
+                    <p>
+                        <a href="{{ url_for('core.edit_profile')}}">
+                            Edit Profile
+                        </a>
+                    </p>
                 {% elif current_user.is_following(user) %}
-                <p><a href="{{ url_for('core.unfollow', username=user.username)}}">Unfollow</a></p>
+                    <div class="follow-link">
+                        <p onclick="unFollowUser()">Unfollow</p>                    
```

Comment:
Please use `button` here just to denote it is clickable and add some css to it. Don't forget to add `cursor:pointer` to css rule.

Reply:
This one is also good to go 👍